api/src/main/resources/org/openmrs/module/commonreports/sql/lab.sql

```diff
@@ -0,0 +1,20 @@
+SELECT 
+sum(CASE WHEN concept_id=(select DISTINCT concept_id from concept_name where name = 'Serial sputum bacilloscopy' and locale='en' and concept_name_type='FULLY_SPECIFIED' and voided = 0) AND value_coded IS NOT NULL THEN 1 ELSE 0 END) as sputum,
```

Comment:
It'd be great if the concepts where provided than hardcoded by name.

Reply:
Done